h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/remote/RemoteXGBoostUploadServlet.java

```diff
@@ -48,7 +48,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
                 destFile = getCheckpointFile(model_key);
             }
             LOG.debug("Saving contents into " + destFile);
-            InputStream is = ServletUtils.extractPartInputStream(request, response);
+            InputStream is = request.getInputStream();
```

Comment:
how is handled the input stream close? could it be in the try-with-resources block?

Reply:
the Request class javadoc does not say anything about the caller having to close the input stream, that is why I elected not to close it, it makes no difference IMO